lib/utils.pm

```diff
@@ -377,6 +377,9 @@ sub ensure_unlocked_desktop {
                 # responsiveness.
                 # open run command prompt (if screen isn't locked)
                 mouse_hide(1);
+                # implementation in backend/driver.pm is QEMU-specific,
+                # so we have to reset pointer on non-QEMU backends.
+                mouse_hide unless check_var('BACKEND', 'qemu');
```

Comment:
I don't get it - this is still mouse_hide

Reply:
border_offset is 0 in this case. but probably it can be rewritten :
`mouse_hide( check_var('BACKEND', 'qemu'))`

I mean directly in line 379 , without adding line 382 ?

Reply:
Well, but this would be the case for all mouse_hides called with 1 - so better ignore it in the backend then.

Reply:
I understand @asmorodskyi's suggestions but it seems to me that we need to move the mouse a bit, that's why `mouse_hide(1)` (and the hide it again on non-QEMU). But I don't understand what @coolo proposes to do in backend... To ignore `border_offset => 1`?

Reply:
well, whenever we call mouse_hide(1) on !qemu, it's bogus - that's what you're saying.

So fix it properly in backend to do whatever you propose to add here